roles/common/tasks/monitoring.yml

```diff
@@ -207,6 +207,13 @@
 - name: log scanning caching dir
   file: path=/var/cache/check-log owner=sensu state=directory
 
+- name: bad memory alert
+  sensu_check: name=bad-memory plugin=check-mcelog.sh use_sudo=true
+               auto_resolve=false interval=20 occurrences=1
+               args='--criticality'
```

Comment:
correct `--criticality {{ common.monitoring.sensu_checks.bad_memory.criticality }}`  or whatever the correct var path is.   should be set to warning or critical.  there's examples of others in the repo doing this.